### ISSUE TITLE
Npm install fail

### ISSUE THREAD

**Issue by Morziloeil** [gemini-testing/hermione#441]
Can someone help me with the following error, at installing Hermione on Mac OS X 10.14 Mojave ?
Ty in advanced.

[2019-09-05T16_24_15_134Z-debug.log](https://github.com/gemini-testing/hermione/files/3580373/2019-09-05T16_24_15_134Z-debug.log)

````
$npm install hermione chai
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> png-img@2.2.0 install /Users/morziloeil/node_modules/png-img
> node-gyp rebuild

  CC(target) Release/obj.target/libpng/third_party/libpng/png.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngerror.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngget.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngmem.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngpread.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngread.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrio.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrtran.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngrutil.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngset.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngtest.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngtrans.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwio.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwrite.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwtran.o
  CC(target) Release/obj.target/libpng/third_party/libpng/pngwutil.o
  LIBTOOL-STATIC Release/png.a
  CC(target) Release/obj.target/zlib/third_party/zlib/adler32.o
  CC(target) Release/obj.target/zlib/third_party/zlib/compress.o
  CC(target) Release/obj.target/zlib/third_party/zlib/crc32.o
  CC(target) Release/obj.target/zlib/third_party/zlib/deflate.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzclose.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzlib.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzread.o
  CC(target) Release/obj.target/zlib/third_party/zlib/gzwrite.o
  CC(target) Release/obj.target/zlib/third_party/zlib/infback.o
  CC(target) Release/obj.target/zlib/third_party/zlib/inffast.o
  CC(target) Release/obj.target/zlib/third_party/zlib/inflate.o
../third_party/zlib/inflate.c:1507:61: warning: shifting a negative signed value is undefined [-Wshift-negative-value]
    if (strm == Z_NULL || strm->state == Z_NULL) return -1L << 16;
                                                        ~~~ ^
1 warning generated.
  CC(target) Release/obj.target/zlib/third_party/zlib/inftrees.o
  CC(target) Release/obj.target/zlib/third_party/zlib/trees.o
  CC(target) Release/obj.target/zlib/third_party/zlib/uncompr.o
  CC(target) Release/obj.target/zlib/third_party/zlib/zutil.o
  LIBTOOL-STATIC Release/zlib.a
  CXX(target) Release/obj.target/png_img/src/init.o
../src/init.cc:10:125: error: too few arguments to function call, single argument 'context' was not specified
    Nan::Set(target, New<String>("PngImg").ToLocalChecked(), New<FunctionTemplate>(PngImgAdapter::NewInstance)->GetFunction());
                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/Users/morziloeil/Library/Caches/node-gyp/12.2.0/include/node/v8.h:5947:3: note: 'GetFunction' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<Function> GetFunction(
  ^
/Users/morziloeil/Library/Caches/node-gyp/12.2.0/include/node/v8config.h:347:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
1 error generated.
make: *** [Release/obj.target/png_img/src/init.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/morziloeil/.nvm/versions/node/v12.2.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:196:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:257:12)
gyp ERR! System Darwin 18.2.0
gyp ERR! command "/Users/morziloeil/.nvm/versions/node/v12.2.0/bin/node" "/Users/morziloeil/.nvm/versions/node/v12.2.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/morziloeil/node_modules/png-img
gyp ERR! node -v v12.2.0
gyp ERR! node-gyp -v v5.0.3
gyp ERR! not ok
npm WARN enoent ENOENT: no such file or directory, open '/Users/morziloeil/package.json'
npm WARN morziloeil No description
npm WARN morziloeil No repository field.
npm WARN morziloeil No README data
npm WARN morziloeil No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! png-img@2.2.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the png-img@2.2.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/morziloeil/.npm/_logs/2019-09-05T16_24_15_134Z-debug.log

**Comment by j0tunn** [comment]
https://github.com/gemini-testing/png-img/issues/45
Use node@8 or node@10 for now. We are working on solution

**Comment by j0tunn** [comment]
fixed in `png-img@2.2.1`
